packages/frameworks/src/frameworks.ts

```diff
@@ -1914,7 +1914,7 @@ export const frameworks = [
         value: 'sanity build',
       },
       devCommand: {
-        value: 'sanity start --port $PORT',
```

Comment:
> But before anyone merges: would changing this retroactively change the dev command for all existing Sanity-detected projects on Vercel? That would break Sanity v2 projects and I'd need to split into Sanity v2 and Sanity v3 frameworks.

We do need to keep these separate. We should make another `sanity-3` entry that can configure v2 (`sanity`) and v3 (`sanity-3`) separately. We just have to decide how the `detectors` can accurately determine the version.

We can detect the `sanity` package version. Is that always going to be in a sanity project?

Reply:
Package version might be more accurate, but just throwing out that you could still use file detection. Sanity 2 uses `sanity.json`, while v3 migrated to `sanity.config.[js|ts|jsx|tsx]`.

There's a way to have the framework detector do conditional commands without creating multiple frameworks?

Reply:
Ahh, I saw that those files were already listed, but that's because [you added them earlier to support v3](https://github.com/vercel/vercel/pull/8540). Yeah, that sounds like a good way to detect the versions.

There is not currently a way to differentiate a command based on version. We'll have to create a new entry here.

Reply:
Yep! That was me, turns out they did change commands after all.

So should the slug `sanity` become `sanity@latest` (currently Sanity 3), and create a new `sanity-2`?

Or keep `sanity` as Sanity v2, and make a new `sanity-3`? That seems to be what Docusaurus did.

Reply:
The original slug should stay the same. It seems that v3 is in error, let's make `sanity-3` for that.